estate/models/estate_property.py

```diff
@@ -0,0 +1,37 @@
+from odoo import fields, models
+from datetime import date, timedelta
+
+default_availability_date = (date.today() + timedelta(days=90)).strftime("%Y-%m-%d")
+
+
```

Comment:
Try using relativeDelta and implement the same in the field directly rather than creating a new variable here
Follow a consistent pattern with spacing and new lines in code.